
# By default, VLP-16 LiDAR is used.
# For other LiDARs, you can modify LiDAR parameters in "sgmtt.utility.h".

kNumSectors: 360                    # Number of angular segments.
kColsPerSector: 5                   # kLidarCols/kNumSectors
kSensorHeight: 1.3                  # Height from ground surface to LiDAR sensor.

# Identify ground.
kGroundSameLineTolerance: 0.035       # 2 degree(0.035), around 0.1m/3m
kGroundSlopeTolerance: 0.182          # 10 degrees(0.182)
kGroundYInterceptTolerance: 0.2       # the intercept(b) of a line.
kGroundPointLineDistThres: 0.1        # Point-Line maximum distance to be classified as ground.

# Identify wall.
kWallSameLineTolerance: 0.1051        # 6 degree(0.1051), 2 degree(0.035)
kWallSlopeTolerance: 3.73             # 85 degrees(11.43), 80 degrees(5.67), 75 degrees(3.73)
kWallLineMinBinNum: 3                 # a wall line should cover at least 3 rings.
kWallPointLineDistThres: 0.1          # Point-Line maximum distance to be classified as wall.
